internal/middleware/enforce_identity.go

```diff
@@ -55,6 +55,11 @@ func EnforceIdentity(next http.Handler) http.Handler {
 
 		topLevelOrgIDFallback(&jsonData)
 
+		logger.Debug().Msgf("Enforcing identity for %s %s (%s)",
+			jsonData.Identity.User.FirstName,
+			jsonData.Identity.User.LastName,
+			jsonData.Identity.User.Email,
+		)
```

Comment:
I was just checking if this would not error out in case of missing User, but it looks like it is okay. Still, if all the values were empty, would this not be confusing? I am not sure though, it might be okay.

Reply:
It is not a pointer so cannot be nil, but you are right that empty values will look strange. Rebasing, let's use the raw JSON. There are no passwords or secret items.